ionic/src/app/components/model/user.ts

```diff
@@ -14,3 +14,10 @@ export interface importDevice{
     updatedAt: string;
     user_id: string;
 }
+export interface addDevice{
```

Comment:
we cannot have 3 typescript for add/delete/list doesn't make sense make one generic